src/DropdownMenu.js

```diff
@@ -72,17 +71,19 @@ class DropdownMenu extends React.Component {
       const position1 = directionPositionMap[this.context.direction] || 'bottom';
       const position2 = right ? 'end' : 'start';
       const poperPlacement = `${position1}-${position2}`;
-      const poperModifiers = !flip ? {
+      const poperModifiers = [
         ...modifiers,
-        ...noFlipModifier,
-      } : modifiers;
-      const popperPositionFixed = !!positionFixed;
+        {
+          name: 'flip',
+          enabled: !!flip,
+        },
+       ];
 
       const popper = (
         <Popper
           placement={poperPlacement}
           modifiers={poperModifiers}
-          positionFixed={popperPositionFixed}
+          strategy={positionFixed ? 'fixed' : undefined}
```

Comment:
Will update the `positionFixed` prop to `strategy` since based on previous reviews, we're welcoming of breaking changes.